.eslintrc

```diff
@@ -85,9 +85,15 @@
     "react/jsx-props-no-spreading": "off",
     "react/state-in-constructor": "off",
     "react/static-property-placement": "off",
+    "arrow-parens": ["error", "always"],
+
+    "no-use-before-define": "off",
+    "@typescript-eslint/no-use-before-define": ["error"],
 
     "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/no-explicit-any": "off"
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/ban-types": "off",
```

Comment:
`@typescript-eslint/ban-types` is complaining about all existing types that are just `Function`.